gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -34,205 +29,30 @@
 from apache_beam.io.iobase import Read
 from apache_beam.transforms import PTransform
 
-__all__ = ['ReadFromVcf', 'ReadAllFromVcf', 'Variant', 'VariantCall',
-           'MalformedVcfRecord']
+from gcp_variant_transforms.beam_io import vcf_parser
 
+# All other modules depend on vcfio for the following const values.
+MalformedVcfRecord = vcf_parser.MalformedVcfRecord
+MISSING_FIELD_VALUE = vcf_parser.MISSING_FIELD_VALUE
+PASS_FILTER = vcf_parser.PASS_FILTER
+END_INFO_KEY = vcf_parser.END_INFO_KEY
+GENOTYPE_FORMAT_KEY = vcf_parser.GENOTYPE_FORMAT_KEY
+PHASESET_FORMAT_KEY = vcf_parser.PHASESET_FORMAT_KEY
+DEFAULT_PHASESET_VALUE = vcf_parser.DEFAULT_PHASESET_VALUE
+MISSING_GENOTYPE_VALUE = vcf_parser.MISSING_GENOTYPE_VALUE
+Variant = vcf_parser.Variant
+VariantCall = vcf_parser.VariantCall
 
-# Stores data about failed VCF record reads. `line` is the text line that
-# caused the failed read and `file_name` is the name of the file that the read
-# failed in.
-MalformedVcfRecord = namedtuple('MalformedVcfRecord',
-                                ['file_name', 'line', 'error'])
-MISSING_FIELD_VALUE = '.'  # Indicates field is missing in VCF record.
-PASS_FILTER = 'PASS'  # Indicates that all filters have been passed.
-END_INFO_KEY = 'END'  # The info key that explicitly specifies end of a record.
-GENOTYPE_FORMAT_KEY = 'GT'  # The genotype format key in a call.
-PHASESET_FORMAT_KEY = 'PS'  # The phaseset format key.
-DEFAULT_PHASESET_VALUE = '*'  # Default phaseset value if call is phased, but
-                              # no 'PS' is present.
-MISSING_GENOTYPE_VALUE = -1  # Genotype to use when '.' is used in GT field.
 
+__all__ = ['ReadFromVcf', 'ReadAllFromVcf', 'MalformedVcfRecord']
```

Comment:
I think you can remove this now (required by Beam, but not by our code). We kept it here for Beam-compatibility, but now we have to change a bunch of stuff anyway, so let's remove it. Ditto for vcf_parser.py.